app/components/candidate_interface/new_reference_send_reminder_component.rb

```diff
@@ -0,0 +1,14 @@
+class CandidateInterface::NewReferenceSendReminderComponent < ViewComponent::Base
+  attr_accessor :reference, :reference_actions_policy
+  delegate :can_send_reminder?, to: :reference_actions_policy
+  delegate :reminder_sent_at, to: :reference
+
+  def initialize(reference)
+    @reference = reference
+    @reference_actions_policy = ReferenceActionsPolicy.new(@reference)
+  end
+
+  def remind_again_at
+    reminder_sent_at + TimeLimitConfig.minimum_hours_between_chaser_emails.hours
```

Comment:
Do we test that this works anywhere? We are incrementing a DateTime with `+` but passing a number of hours?
In a rails console

```ruby
4] pry(main)> now = DateTime.now
=> Thu, 11 Aug 2022 15:25:52 +0100
[5] pry(main)> now + 1
=> Fri, 12 Aug 2022 15:25:52 +0100
```

Also https://github.com/DFE-Digital/apply-for-teacher-training/blob/98a06ead3753c6bad40554e22a8f77c64f34f4a7/app/lib/time_limit_config.rb#L24 
`TimeLimitConfig.minimum_hours_between_chaser_emails` is an integer and doesn't implement an `hours` method

Reply:
Interesting 🤔 The spec does passes. Let me have a look

Reply:
It works because the `remind_again_at` is a `ActiveSupport::TimeWithZone` instance and the hours is an `ActiveSupport::Duration` so that's why the page renders the date and time correctly and [the spec](https://github.com/DFE-Digital/apply-for-teacher-training/pull/7296/files#diff-c59d409d2cb537b1b311be4df58966f262f4fc81777f004a98574dcc338054a8R27) passes.

Reply:
Ugh `48.hours` in Rails is `48 hours`, total voodoo. But I agree it works.